client/components/domains/example-domain-suggestions/style.scss

```diff
@@ -16,7 +18,7 @@
 		text-decoration: underline;
 	}
 
-	@include breakpoint( "<660px" ) {
+	@include breakpoint( '<660px' ) {
```

Comment:
Why is this scss file changing double quotes to single quotes while the others are doing the opposite?

Reply:
This was me fighting with prettier formatting .scss files--converted back to `"`, which seems to be what's usually used.

Reply:
It's so helpful until it's not :)